assignment09/part01/process.php

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 9, Part 1</title>
+</head>
+<body>
+
+<?php
+    $weight = $_POST['weight'];
+    $height = $_POST['height'];
+
+    //formula for calculating BMI:
+    //BMI = (Weight in Pounds / (Height in inches x Height in inches)) x 703
+    //source: http://www.bmi-calculator.net/bmi-formula.php
+    
+    $BMI_VALUE = ($weight / ($height * $height)) * 703;
+
+    if ($BMI_VALUE < 18) {
+        $LABEL_VALUE = 'underweight';
+    } else if ($BMI_VALUE > 30) {
+        $LABEL_VALUE = 'obese';
+    } else {
+        $LABEL_VALUE = 'normal';
+    }

```

Comment:
good if/else! I like that you left the middle scenario until the else - that is the most efficient 👍